packages/mds-stream-processor/connectors/kafka-connector.ts

```diff
@@ -80,7 +81,130 @@ export const KafkaSource =
           return messageCounter.increment()
         }
       },
+      undefined,
+      options
+    )
+  }
+
+const asChunks = <TMessage>(messages: TMessage[], size = 1000) => {
```

Comment:
Can we put this as a helper method in `mds-utils` and re-use it in `mds-repository`? This is largely copy-pasted from mds-repository